py/server/deephaven/jcompat.py

```diff
@@ -72,6 +72,13 @@ def j_map_to_dict(m) -> Dict[Any, Any]:
     return {e.getKey(): wrap_j_object(e.getValue()) for e in m.entrySet().toArray()}
 
 
+def j_list_to_list(jlist) -> List[Any]:
+    """Converts a java list to a python list."""
+    if not jlist:
+        return None
+
```

Comment:
return [] is better so that callers won't get tripped on ```len(j_list_to_list(...)```, even though in Java null and [] can mean very different things. Also the type hint of List[Any] basically says it is always a List.

Reply:
Forgot to change that one, sry, ty.